board/crosscompile-toolchain.cmake

```diff
@@ -0,0 +1,41 @@
+## This file handles cross-compilation configurations for aarch64,
+## known as arm64. The objective of this file is to find and assign
+## cross-compiler and the entire toolchain.
+## It works best with buildroot toolchain, when using it the user
```

Comment:
It could be useful to add a link to the buildroot website here.  (I didn't know what buildroot was before we started on these PRs.)